src/main/webapp/app/teams/teams-skills.scss

```diff
@@ -50,7 +51,7 @@
             padding: 5px 5px 5px 20px;
             word-break: break-word;
 
-            &.irrelevant {
+            &.IRRELEVANT {
```

Comment:
This looks odd. All caps class name? Is this needed because you use the ENUM value of the backend as a classname?

Reply:
Refactored to use better CSS class names.